src/components/Plan.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import '../index.css'
+
+const Plan = ({ title1, title2, plans, buttonText, finishingText, onClick }) => {
```

Comment:
This is a good start, just some styling and flexbox or grid needed to complete the design :-)